src/sass/patterns/_data-tables.scss

```diff
@@ -0,0 +1,5 @@
+.slds-table_header-hidden {
+  thead {
+    display: none;
+  }
+}
```

Comment:
@stacyk I don't love this, but...

WHEREAS... React-SLDS does not yet have an implementation for the SLDS headless DataTable (https://lightningdesignsystem.com/components/data-tables/#Headless). So we have to override if we don't want the `thead` visible.

WHEREAS... I first tried importing and using the SLDS `assistive-text` mixin (https://github.com/salesforce-ux/design-system/blob/21b01368fc63db0cf53c077def1ebffd9bb4c64d/ui/dependencies/_visibility.scss#L6-L17), but the absolute positioning broke the `fixedLayout` DataTable.

THEREFORE... Feel free to clean this up. The reason I used `fixedLayout` was to truncate the commit message column. If you want to remove that and use the `assistive-text` mixin, maybe you have a better way to make truncation work.

Reply:
I removed the fixed table layout so maybe this assistive text mixin would work now?